File: spotlight/src/main/java/com/takusemba/spotlight/OnSpotlightStateChangedListener.java
package com.takusemba.spotlight;

/**
 * On Spotlight Ended Listener
 **/
public interface OnSpotlightStateChangedListener {

    /**
     * Called when Spotlight is started
     */
    void onStarted();

    /**
     * Called when Spotlight is ended
     */
    void onEnded();
}

File: spotlight/src/main/java/com/takusemba/spotlight/OnSpotlightStateChangedListener.kt
package com.takusemba.spotlight

/**
 * On Spotlight Ended Listener
 */
interface OnSpotlightStateChangedListener {

  /**
   * Called when Spotlight is started
   */
  fun onStarted()

  /**
   * Called when Spotlight is ended
   */
  fun onEnded()
}
